src/compas_rhino/geometry/brep/__init__.py

```diff
@@ -0,0 +1,13 @@
+from compas.plugins import plugin
+
+from .brep import RhinoBRep
+
+
+@plugin(category="factories", requires=["Rhino"])
+def new_brep_from_shape(cls, *args, **kwargs):
+    return RhinoBRep.from_shape(*args, **kwargs)
```

Comment:
I agree. May I suggest we drop verboseness a notch and just call them `from_x`? otherwise we're looking at a `new_brep_from_brep`..

Also, I'd keep a `new_brep` plugin which would do something like `RhinoBRep.from_brep(Rhino.Geometry.Brep())` to essentially create an empty `Brep` which can be filled later on. This will allow:
```python3
from compas.geometry import BRep
b = BRep()
```
And in the future we could offer adding faces etc. What do you think?

And since we're discussing names, I went with `BRep` cause of `compas_occ` but I like `Brep` much better. What would you say about changing?

Reply:
well and all the other stuff sounds good as well :)